From 9031cc457a75912fffa4925fd8a7b19768a36395 Mon Sep 17 00:00:00 2001
From: Meriipu <Meriipu@users.noreply.github.com>
Date: Mon, 12 Oct 2020 03:23:31 +0200
Subject: [PATCH 9/9] Remove inspect accessibility from context menu

It only gets in my way when I accidentally click it instead of clicking
inspect element, and it can be accessed from the accessibility tab in
devtools if needed (I have yet to need it enough for it to have its own
entry in the context menu).
---
 browser/base/content/browser-context.inc | 4 ----
 1 file changed, 4 deletions(-)

diff --git a/browser/base/content/browser-context.inc b/browser/base/content/browser-context.inc
index 58de53ac54af..02208fcad0e1 100644
--- a/browser/base/content/browser-context.inc
+++ b/browser/base/content/browser-context.inc
@@ -346,10 +346,6 @@
                 data-l10n-id="main-context-menu-bidi-switch-page"
                 oncommand="gContextMenu.switchPageDirection();"/>
       <menuseparator id="inspect-separator" hidden="true"/>
-      <menuitem id="context-inspect-a11y"
-                hidden="true"
-                data-l10n-id="main-context-menu-inspect-a11y-properties"
-                oncommand="gContextMenu.inspectA11Y();"/>
       <menuitem id="context-inspect"
                 hidden="true"
                 data-l10n-id="main-context-menu-inspect-element"
-- 
2.28.0

